### ISSUE TITLE
SRT does not work with : OBS 26.1.2-197 

### ISSUE THREAD

**Issue by FilipStadler** [obsproject/obs-studio#4262]
CPU Name: Intel(R) Core(TM) i7 CPU         920  @ 2.67GHz
CPU Speed: 2680.556MHz
Physical Cores: 4, Logical Cores: 8
Physical Memory: 16009MB Total, 7028MB Free
Kernel Version: Linux 5.8.0-43-generic
Distribution: "Ubuntu" "20.04"
Session Type: x11
Window System: X11.0, Vendor: The X.Org Foundation, Version: 1.20.9
Portable mode: false
OBS 26.1.2-197-g81a89e689 (linux)

I keep get MP: Failed to open media: 'srt://192.168.10.100:4003?mode=listener' in the OBS log file.
No problem using udp:// just srt does not work  - I have compiled the ffmpeg using  https://github.com/markus-perl/ffmpeg-build-script so I sure srt is enabled and also working using srt-live-transmit 

17:39:05.805: [Media Source 'Media Source srt-test']: settings:
17:39:05.805: 	input:                   srt://192.168.10.100:4003?mode=listener
17:39:05.805: 	input_format:            mpegts
17:39:05.805: 	speed:                   100
17:39:05.805: 	is_looping:              no
17:39:05.805: MP: Failed to open media: 'srt://192.168.10.100:4003?mode=listener'
17:39:05.805: 	is_hw_decoding:          no
17:39:05.805: 	is_clear_on_media_end:   yes
17:39:05.805: 	restart_on_activate:     yes
17:39:05.805: 	close_when_inactive:     no

Does OBS include SRT support or is it missing - I have compiled srt and also the ubuntu ffmpeg is replaced - OBS is compiled but I keep getting OBS log file says  MP: Failed to open media: 'srt://192.168.10.100:4003&mode=listener' and it does not open sockets and listen if I replace the srt with udp it does open and listen so something is not working.  I am not a programmer but I would say it looks like a bug with the API between OBS and the SRT 

does anybody know where to look to make sure SRT is a option protocol because I have tried alot to solve the problem.  I know the SRT is working using srt-live-transmit but its like the connection between OBS and SRT is missing ?

It does not show up with netstat -au
udp        0      0 0.0.0.0:4003            0.0.0.0:*

**Comment by tytan652** [comment]
Could you specify if you build obs yourself or using a snap or a deb package from a ppa ?

**Comment by kkartaltepe** [comment]
you need a system ffmpeg with support for srt.

**Comment by FilipStadler** [comment]
My FFMpeg does include SRT and and I have at stated I have compiled  OBS 26.1.2-197 so it should be working.

This is the bug or problem that OBS does not work with SRT - I have tried with the build from PAA and and also
snap but because they both where not working so I recompiled it all and got the same problem.

 ffmpeg is with support for srt -  --enable-libsrt  
 
 I have removed ffmpeg and OBS and recompiled it all - this is very strange and its the only bug I have.
 
 srt://192.168.10.100:4003&mode=listener' does not listen but udp://@:4003 would work fine.
